source/community/reactnative/src/utils/host-config.js

```diff
@@ -516,6 +516,7 @@ export class HostConfig {
 	factSet = new FactSetConfig();
 	fontStyles = new FontStyleConfig();
 
+	hostCapabilities = new HostCapabilities()
```

Comment:
Just a question - So by `default`, we are not giving any `capabilities` right?

Reply:
Thats correct. capabilities need to be set in the hostConfig and if the items in the 'requires' object matches the host capabilities, it will be rendered. Else it will fallback.